hasher-matcher-actioner/webapp/src/Api.jsx

```diff
@@ -222,7 +221,7 @@ export function fetchHashCount() {
 }
 
 export function fetchAllActions() {
-  return apiGet('/actions/');
```

Comment:
What's this change? Was this broken before?

Reply:
Yes, there were two slashes between the domain name / port and the beginning of the route (e.g. https://8mrupgnwyf.execute-api.us-east-1.amazonaws.com//actions/)

Created https://github.com/facebook/ThreatExchange/issues/615 to track cleaning this up systematically.